modules/account/src/main/kotlin/edu/stanford/spezi/module/account/register/RegisterValidation.kt

```diff
@@ -0,0 +1,29 @@
+package edu.stanford.spezi.module.account.register
+
+import java.time.LocalDate
+
+fun isEmailValid(email: String): Boolean =
+    android.util.Patterns.EMAIL_ADDRESS.matcher(email).matches()
+
+fun isPasswordValid(password: String): Boolean =
+    password.length >= 6 // Minimum for firebase
+
+fun isFirstNameValid(firstName: String): Boolean =
+    firstName.isNotEmpty()
+
+fun isLastNameValid(lastName: String): Boolean =
+    lastName.isNotEmpty()
+
+fun isGenderValid(gender: String): Boolean =
```

Comment:
Might not be needed if we instead introduce an enum for the genders and display it in dropdown element as you are doing. Instead of using `TextFieldUpdate` action, introduce a `GenderSelected(GenderType)` action instead